units/provisioning_setup_host.go

```diff
@@ -801,12 +801,12 @@ func mountLinuxVolume(ctx context.Context, env evergreen.Environment, h *host.Ho
 	}()
 
 	output, err := h.RunJasperProcess(ctx, env, &options.Create{
-		Args: []string{"lsblk"},
+		Args: []string{"lsblk", "-o", "+UUID"},
 	})
 	if err != nil {
 		return errors.Wrap(err, "can't run lsblk")
 	}
-	deviceName, err := getMostRecentlyAddedDevice(output)
+	deviceName, deviceUUID, err := getMostRecentlyAddedDevice(output)
```

Comment:
is the a circumstance where we need the device name anymore? 

you can do cmd.AppendArgs("umount", "-f", deviceName) below.

Reply:
I left the device name there for when we [check if there's already a filesystem](https://github.com/evergreen-ci/evergreen/blob/a4ed7498e9b97be9a007f82ee95a5d89d29c2640/units/provisioning_setup_host.go#L828)
I thought about using the UUID for mount and umount, but they seemed equivalent, and once I still needed the device name for something else I just left it.

Reply:
nvm: I think the check can use the FSType reported by lsblk 😃

Reply:
I still needed the device name for the case where the filesystem wasn't yet created